relay-ourlogs/src/ourlog.rs

```diff
@@ -70,9 +72,152 @@ pub fn otel_to_sentry_log(otel_log: OtelLog) -> OurLog {
     }
 }
 
+/// Transform event breadcrumbs to OurLogs.
+///
+/// Only converts up to `max_breadcrumbs` breadcrumbs.
+pub fn breadcrumbs_to_ourlogs(event: &Event, max_breadcrumbs: usize) -> Vec<OurLog> {
+    let event_trace_id = event
+        .context::<TraceContext>()
+        .and_then(|trace_ctx| trace_ctx.trace_id.value())
+        .cloned();
+
+    let breadcrumbs = match event.breadcrumbs.value() {
+        Some(breadcrumbs) => breadcrumbs,
+        None => return Vec::new(),
+    };
```

Comment:
nit: You could change the signature to `Option<Vec<OurLog>>` and shortcut with `.value()?` here (and in other places below).